src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -389,89 +389,89 @@ public abstract partial class WebRequestPSCmdlet : PSCmdlet
         internal virtual void ValidateParameters()
         {
             // sessions
-            if (WebSession != null && SessionVariable != null)
+            if (WebSession is not null && SessionVariable is not null)
             {
                 ErrorRecord error = GetValidationError(WebCmdletStrings.SessionConflict, "WebCmdletSessionConflictException");
                 ThrowTerminatingError(error);
             }
 
             // Authentication
-            if (UseDefaultCredentials && Authentication != WebAuthenticationType.None)
+            if (UseDefaultCredentials && Authentication is not WebAuthenticationType.None)
             {
                 ErrorRecord error = GetValidationError(WebCmdletStrings.AuthenticationConflict, "WebCmdletAuthenticationConflictException");
                 ThrowTerminatingError(error);
             }
 
-            if (Authentication != WebAuthenticationType.None && Token != null && Credential != null)
+            if (Authentication is not WebAuthenticationType.None && Token is not null && Credential is not null)
```

Comment:
Please revert patterns like `Authentication is not WebAuthenticationType.None`.

Reply:
Or better move this in follow PR after we merge that.

Reply:
Done